src/SeleniumLibrary/keywords/screenshot.py

```diff
@@ -241,16 +241,64 @@ def _embed_to_log_as_file(self, path, width):
     @keyword
     # def print_page_as_pdf(self, print_options: Optional[PrintOptions]=None,):
     # def print_page_as_pdf(self, print_options: Optional[Union[PrintOptions, dict]]=None):
-    def print_page_as_pdf(self, print_options: Union[PrintOptions, dict, None]=None):
+    # def print_page_as_pdf(self, print_options: Union[PrintOptions, dict, None]=None):
```

Comment:
comments can be removed

Reply:
Removed. Added some more, and eventually removed those as well. [Laughing]